src/EditorFeatures/Core.Wpf/Options/EditorConfigDocumentOptionsProvider.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace Microsoft.CodeAnalysis.Editor.Options
 {
-    // This class is currently linked into both EditorFeatures.Wpf (VS in-process) and RemoteWorkspaces (Roslyn out-of-process).
+    // This class is currently linked into EditorFeatures.Wpf (VS in-process), RemoteWorkspaces (Roslyn out-of-process), and MSBuild Workspaces.
```

Comment:
this still seems problematic.  We shoudl not be linking in Editor code into the Workspace layer.

Reply:
This issue still applies.